cpu/stm32l0/periph/adc.c

```diff
@@ -0,0 +1,184 @@
+/*
+ * Copyright (C) 2014-2017 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     cpu_stm32l0
+ * @ingroup     drivers_periph_adc
+ * @{
+ *
+ * @file
+ * @brief       Low-level ADC driver implementation
+ *
+ * @author      Aurélien Fillau <aurelien.fillau@we-sens.com>
+ *
+ * @}
+ */
+
+#include "cpu.h"
+#include "mutex.h"
+#include "periph/adc.h"
+
+#ifdef ADC_CONFIG
```

Comment:
Here it shoud be:
```c
#ifdef ADC_NUMOF
```

Reply:
Are you sure about this ? Because it is done like this everywhere else.

Reply:
> Are you sure about this ?

Yes

> Because it is done like this everywhere else

For ADC but not for the other peripheral drivers (look at the stm32-common ones. And there's #7277 that will use the same pattern once merged (see this [line](https://github.com/RIOT-OS/RIOT/pull/7277/files#diff-fb6eee5fcf3447047d0ff0a74b016643R28).

Reply:
done